js/libs/form.js

```diff
@@ -1013,6 +1013,14 @@ Fliplet.FormBuilder.get = function (name) {
   });
 };
 
-Fliplet.FormBuilder.getAll = function () {
-  return Promise.all(formBuilderInstances);
+Fliplet.FormBuilder.getAll = function (name) {
+  return Promise.all(formBuilderInstances).then(function(forms) {
+    if (typeof name === 'undefined') {
+      return forms
+    }
+
+    return forms.filter(function(vueForm) {
```

Comment:
I don't think it's necessary to have the `vue` prefix here

Reply:
I don't think so either. I was following the previous pattern. I'll remove the prefix.